### LIBRERIAS

In [2]:
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import json
import ast
import warnings
import pyarrow as pa
import pyarrow.parquet as pq
warnings.filterwarnings('ignore')

In [3]:
# Se carga el archivo para generar un dataframe 
users_items = pd.read_csv('csv_limpios/user_items.csv')
users_items

user_id  items_count           steam_id      id  \
0        76561197970982479          277  76561197970982479      10   
1        76561197970982479          277  76561197970982479      20   
2        76561197970982479          277  76561197970982479      30   
3        76561197970982479          277  76561197970982479      40   
4        76561197970982479          277  76561197970982479      50   
...                    ...          ...                ...     ...   
5153204  76561198329548331            7  76561198329548331  346330   
5153205  76561198329548331            7  76561198329548331  373330   
5153206  76561198329548331            7  76561198329548331  388490   
5153207  76561198329548331            7  76561198329548331  521570   
5153208  76561198329548331            7  76561198329548331  519140   

                             item_name  playtime_forever  playtime_2weeks  
0                       Counter-Strike               6.0              0.0  
1                Team Fortress Classic               0.0              0.0  
2                        Day of Defeat               7.0              0.0  
3                   Deathmatch Classic               0.0              0.0  
4            Half-Life: Opposing Force               0.0              0.0  
...                                ...               ...              ...  
5153204                   BrainBread 2               0.0              0.0  
5153205                    All Is Dust               0.0              0.0  
5153206  One Way To Die: Steam Edition               3.0              3.0  
5153207          You Have 10 Seconds 2               4.0              4.0  
5153208                     Minds Eyes               3.0              3.0  

[5153209 rows x 7 columns]

In [4]:
users_items.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 5153209 entries, 0 to 5153208
Data columns (total 7 columns):
 #   Column            Dtype  
---  ------            -----  
 0   user_id           object 
 1   items_count       int64  
 2   steam_id          int64  
 3   id                int64  
 4   item_name         object 
 5   playtime_forever  float64
 6   playtime_2weeks   float64
dtypes: float64(2), int64(3), object(2)
memory usage: 275.2+ MB


El dataset 'users_items' posee 7 columnas y 5153209  registros.

El contenido de las columnas es el siguiente:
- user_id: elemento identificador de cada usuario que posee un item (titulo del dataset 'steam_games'). Dato tipo str
- items_count: cantidad de items que posee cada usuario. Dato tipo int
- steam_id: elemento identificador interno de la plataforma steam de cada usuario. Dato tipo int
- id: elemento identificador de cada titulo registrado en el dataset. Dato tipo int
- item_name: titulo del juego o programa. Dato tipo str
- playtime_forever: tiempo acumulado de uso del item por usuario.
- playtime_2weeks:  tiempo acumulado de uso del item por usuario de las ultimas 2 semanas.

In [5]:
# Se verifica cantidad de nulos por columna
users_items.isnull().sum()

user_id             0
items_count         0
steam_id            0
id                  0
item_name           0
playtime_forever    0
playtime_2weeks     0
dtype: int64

In [6]:
# Se verifica la cantidad de datos unicos por columna
users_items.nunique()

user_id             70912
items_count           924
steam_id            70912
id                  10978
item_name           10947
playtime_forever    48861
playtime_2weeks      4450
dtype: int64